app/googleanalytics/ga4.py

```diff
@@ -3,72 +3,36 @@
 """
 
 import os
+import uuid
 
-from flask import current_app, Blueprint, request, send_file
-from jsonschema import FormatChecker, ValidationError
-from jsonschema.validators import Draft202012Validator
+from flask import current_app, Blueprint, send_file
 
-from app.googleanalytics.ga4_schemas import ga4_request_schema
 from app.celery.process_ga4_measurement_tasks import post_to_ga4
 
 ga4_blueprint = Blueprint('ga4', __name__, url_prefix='/ga4')
 
-ga4_request_validator = Draft202012Validator(ga4_request_schema, format_checker=FormatChecker(['uuid']))
-
 GA4_PIXEL_TRACKING_IMAGE_PATH = f'{os.getcwd()}/images/ga4_pixel_tracking.png'
 
 
-@ga4_blueprint.route('/open-email-tracking', methods=['GET'])
-def get_ga4():
+@ga4_blueprint.route('/open-email-tracking/<notification>', methods=['GET'])
+def get_ga4(notification):
     """
     This route is used for pixel tracking.  It is exercised when a veteran opens an e-mail.
     The route returns a pixel image to avoid a broken icon image in notification emails.
     """
-    # https://flask.palletsprojects.com/en/3.0.x/api/#flask.Request.args
-    url_parameters_dict = request.args.to_dict()
-    current_app.logger.debug('GA4 url params: %s', url_parameters_dict)
-
-    # This could raise ValidationError.
-    ga4_request_validator.validate(url_parameters_dict)
-
-    template_name = url_parameters_dict['campaign']
-    template_id = url_parameters_dict['campaign_id']
-    name = url_parameters_dict['name']
-    source = url_parameters_dict['source']
-    medium = url_parameters_dict['medium']
-    service_name, service_id, notification_id = url_parameters_dict['content'].split('/')
-
-    current_app.logger.info(
-        'GA4: post_to_ga4: template_name: %s, template_id: %s, service_name: %s, service_id: %s, notification_id: %s',
-        template_name,
-        template_id,
-        service_name,
-        service_id,
-        notification_id,
-    )
-    post_to_ga4.delay(
-        notification_id,
-        template_name,
-        template_id,
-        service_id,
-        service_name,
-        name=name,
-        source=source,
-        medium=medium,
-    )
+    current_app.logger.debug('GA4 email_opened for notification: %s', notification)
+
+    # Verify that notification is a valid uuid4
+    try:
+        uuid.UUID(notification, version=4)
+    except ValueError:
+        current_app.logger.error('GA4: Invalid notification ID %s', notification)
+    else:
+        post_to_ga4.delay(
+            notification,
+            current_app.config['GA4_PIXEL_TRACKING_NAME'],
```

Comment:
Oh - Are the config options being passed in here because celery does not have access to those options? 

Like I would think you'd call `current_app.config['whatever']` in the POST to GA4 instead of passing them thru as an arg to POST GA4  in this method.